defaults/openstack/ha.pan

```diff
@@ -0,0 +1,39 @@
+template defaults/openstack/ha;
+
+variable OS_GLANCE_PORT = 9292;
```

Comment:
I think it's a better thing to use ?= operator instead of = and keep variable definition on config template.